lib/build_cache.py

```diff
@@ -243,6 +243,8 @@ class File_Metadata:
    #   hardlink_to ... If non-None, file is a hard link to this other file,
    #                   stored as a Path object relative to the image root.
    #
+   #   xattrs ........ Extended attributes of the file, stored as a dictionary.
```

Comment:
Also say what the keys and values are. (In particular, readers might not know anything about how xattrs work.)